kv/mpp.go

```diff
@@ -30,19 +30,25 @@ type MPPTaskMeta interface {
 
 // MPPTask means the minimum execution unit of a mpp computation job.
 type MPPTask struct {
-	Meta    MPPTaskMeta // on which store this task will execute
-	ID      int64       // mppTaskID
-	StartTs uint64
-	TableID int64 // physical table id
+	Meta         MPPTaskMeta // on which store this task will execute
+	ID           int64       // mppTaskID
+	StartTs      uint64
+	QueryTs      uint64 // timestamp of query execution, used for TiFlash minTSO schedule
+	LocalQueryID uint64 // unique mpp query id in local tidb memory.
+	ServerID     uint64
+	TableID      int64 // physical table id
```

Comment:
`gather_id` is not considered?

Reply:
> `gather_id` is not considered?

`gather_id` is not relevant with stale_read, so not considered in this PR.